acceptance-tests/spec/email_with_pdf_spec.rb

```diff
@@ -48,6 +52,10 @@
     fill_in 'page_autocomplete--autocomplete_autocomplete', with: "California Spangled\n" # the new line "presses enter" on the selected option
     continue
 
+    # upload
+    # attach_file("upload[1]", 'spec/fixtures/files/hello_world.txt')
+    continue
```

Comment:
As we have time left on this card I would like fully test email submissions due to its importance.